charts/datadog/templates/agent-cilium-network-policy.yaml

```diff
@@ -50,12 +68,16 @@ specs:
         {{- end }}
     egress:
       - toFQDNs:
-          {{- if $.Values.datadog.dd_url}}
-          - matchName: {{ trimPrefix "https://" $.Values.datadog.dd_url }}
```

Comment:
The `datadog.dd_url` parameter still exists: https://github.com/DataDog/helm-charts/blob/9611b4af7d929b7eb8792fd3e8ee347cb5bacc8c/charts/datadog/values.yaml#L67
We should support both `datadog.dd_url` and `datadog.site`.